util/util.go

```diff
@@ -11,7 +11,8 @@ import (
 	"github.com/textileio/go-textile-core/thread"
 )
 
-func CreateLog(p peer.ID) (info thread.LogInfo, err error) {
+// CreateLog creates a new log with the given peer as host.
+func CreateLog(host peer.ID) (info thread.LogInfo, err error) {
```

Comment:
`peer.IDFrompublicKey` has a `deprecated` annotation.

Reply:
oh, good catch - the wrong `peer` package is imported. will fix